src/main/java/uk/gov/hmcts/darts/common/config/security/SecurityConfig.java

```diff
@@ -51,7 +50,6 @@ public class SecurityConfig {
     private final InternalAuthProviderConfigurationProperties internalAuthProviderConfigurationProperties;
 
     @Bean
-    @Order(1)
```

Comment:
This is a different annotation than the others removed - `org.junit.jupiter.api.Order`
Comment for awareness - hopefully it's ok to be removed

Reply:
Good point but I don't believe this will have a negative impact. We need a very good reason to order. We should heavily document any ordering that is required.